packages/babel-parser/src/parser/statement.js

```diff
@@ -1924,6 +1924,12 @@ export default class StatementParser extends ExpressionParser {
       }
     }
 
+    const assertions = this.maybeParseImportAssertions();
+    if (assertions) {
+      // $FlowIgnore
+      node.assertions = assertions;
```

Comment:
Please add type information to `packages/babel-parser/src/types.js`.

Reply:
fa83ccb :+1: